assets/stylesheets/directly.scss

```diff
@@ -0,0 +1,503 @@
+// Directly's Real Time Messaging (RTM) widget is an on-demand customer support tool.
+// Read more about the RTM widget and how it's used in Calypso in the `lib/directly` README.
+//
+// The RTM widget loads in an iframe, so we can't adjust the widget's internal styles
+// with the primary Calypso stylesheet. This file is bundled as a separate CSS file to be
+// linked from inside the widget's iframe.
+//
+// We have two Directly builds — Production and Sandbox. Directly requires that all
+// custom CSS files exist at a public web address, and these can't change on-the-fly
+// based on the Calypso environment. So the CSS files Directly uses are hard-set to:
+//
+// Sandbox:    https://wpcalypso.wordpress.com/calypso/directly.css
+// Production: https://wordpress.com/calypso/directly.css
+//
+// ## DEVELOPMENT ENVIRONMENT INSTRUCTIONS:
+//
+// Calypso's development environment uses the Sandbox Directly build, which requests custom
+// CSS from the wpcalypso server. This means that to see local CSS changes you'll need to
+// make your browser rewrite those CSS URLs to the local version:
+//
+// https://wpcalypso.wordpress.com/calypso/directly.css -> http://calypso.localhost:3000/calypso/directly.css
+//
+// You can accomplish this using a browser extension such as Switcharoo Redirector
+// for Chrome: https://chrome.google.com/webstore/detail/switcheroo-redirector/cnmciclhnghalnpfhhleggldniplelbg
+//
+// An unfortunate side-effect is that the widget's iframe src is https, but this local
+// CSS file is http. So most browsers will block the CSS by default to prevent mixed
+// content. In order to see the custom CSS you will need to temporarily allow insecure
+// content on the page. In Chrome for Mac you can issue this Terminal command:
+//
+// open -a Google\ Chrome --args --allow-running-insecure-content
+//
+// QUIT AND RESTART YOUR BROWSER NORMALLY WHEN YOU'RE DONE MAKING LOCAL CSS CHANGES.
+
+@import 'shared/mixins/mixins';
+@import 'shared/colors';
+@import 'shared/forms';
+@import 'shared/typography';
+
+* {
+	font-family: $sans;
+}
+
+body {
+	color: $gray-dark;
+	-webkit-font-smoothing: subpixel-antialiased;
+}
+
+html,
+input,
+select,
+textarea {
+	font-family: $sans;
+}
+
+// Main button
+.btn {
+	background: $blue-medium;
+	border: 1px solid darken( $blue-medium, 8% );
+	color: $white;
+	border-width: 1px 1px 2px;
+	cursor: pointer;
+	font-size: 14px;
+	font-weight: 500;
+	vertical-align: top;
+	box-sizing: border-box;
+	line-height: 21px;
+	border-radius: 4px;
+	padding: 7px 14px 9px;
+	-webkit-appearance: none;
+	-moz-appearance: none;
```

Comment:
We're using an autoprefixer so I think we don't need to explicitly put `-moz` and `-webkit` here.